### PR TITLE
README overhaul

### PR DESCRIPTION
**Pull request by aguerra**
- added coverage
- rewrite based on new structure

---
This change is [Reviewable](https://reviewable.io/reviews/luizalabs/teresa-api/258)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,141 +1,82 @@
...
-    $ teresa login --user myuser@mydomain.com
 
-Create a new team (optional, requires admin privileges):
+This creates a new cluster called `mycluster` and sets it as the current one.
+Log in and create another user and a team:
 
-    $ teresa team create myteam
-    $ teresa team add-user --team myteam --user myuser@mydomain.com
+    $ teresa login --user admin_email
+    $ teresa team create --email myemail myteam
```
Comment (drgarcia1986) [inline README.md]: maybe:
```diff
- $ teresa team create --email myemail myteam
+ $ teresa team create myteam --email myemail
```
💅

File: README.md [hunk 2]
```diff
@@ -1,141 +1,82 @@
...
+
+Teresa has the concept of teams, which are just sets of users. An application
+belongs to a team and all its users can perform all operations on it. There are
+administrative users, which are just regular users with an admin flag set up and
+only them can do user and team management.
+
+To create an admin user you need access to the environment where the Teresa
+server is running (often a Kubernetes POD):
+
+    $ teresa-server create-super-user --email admin_email --password xxxxxxxx
```
Comment (yagonobre) [inline README.md]: Maybe 
```
export POD_NAME=$(kubectl get pods --namespace teresa -l "app=teresa" -o jsonpath="{.items[0].metadata.name}")
kubectl exec $POD_NAME -it  teresa-server create-super-user --email admin_email --password xxxxxxxx -n teresa
```

File: README.md [hunk 3]
```diff
@@ -1,141 +1,82 @@
...
+
+Now you can start creating other users and teams. First, you need to get the
+Teresa endpoint created by Kubernetes and configure the client (get it
+[here](https://github.com/luizalabs/teresa-api/releases/latest)):
 
     $ teresa config set-cluster mycluster --server <teresa-endpoint>
     $ teresa config use-cluster mycluster
```
Comment (drgarcia1986) [inline README.md]: In the first `set-cluster` command, Teresa will set the cluster as the current cluster.
Maybe you can simplify to using only one command.

**Comment by aguerra** [comment]
@drgarcia1986 @yagonobre @arnaldopereira updated.

File: README.md [hunk 4]
```diff
@@ -1,141 +1,82 @@
...
-
-## CLI
-
-Build the CLI:
-
-    $ make build-client
-
-Steps to setup a new cluster and deploy a new application, assuming you already have the API running:
+    $ export TERESA_RSA_PUBLIC=`cat teresa.rsa.pub | base64`
+    $ helm add repo luizalabs http://helm.k8s.magazineluiza.com
```
Comment (yagonobre) [inline README.md]: please change to `helm repo add` i made a typo in helm readme

**Pull request merged**